tests/e2e/tests/devfiles/JavaSpringBoot.spec.ts

```diff
@@ -0,0 +1,60 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import 'reflect-metadata';
+import { WorkspaceNameHandler} from '../..';
+import * as projectAndFileTests from '../../testsLibrary/ProjectAndFileTests';
+import * as commonLsTests from '../../testsLibrary/LsTests';
+import * as workspaceHandling from '../../testsLibrary/WorksapceHandlingTests';
+import * as codeExecutionTests from '../../testsLibrary/CodeExecutionTests';
+
+const stack : string = 'Java Spring Boot';
+const workspaceSampleName: string = 'java-web-spring';
+const workspaceRootFolderName: string = 'src';
+const fileFolderPath: string = `${workspaceSampleName}/${workspaceRootFolderName}/main/java/org/springframework/samples/petclinic`;
+const tabTitle: string = 'PetClinicApplication.java';
+const codeNavigationClassName: string = 'SpringApplication.class';
+const taskName: string = 'maven build';
+
+suite(`${stack} test`, async () => {
+    suite (`Create ${stack} workspace`, async () => {
+        workspaceHandling.createAndOpenWorkspace(stack);
+        projectAndFileTests.waitWorkspaceReadiness(workspaceSampleName, workspaceRootFolderName);
+    });
+
+    suite('Test opening file', async () => {
+        // opening file that soon should give time for LS to initialize
+        projectAndFileTests.openFile(fileFolderPath, tabTitle);
+    });
+
+    suite('Validation of workspace build and run', async () => {
```

Comment:
It says it's testing build and run, but only the build command is executed.

Reply:
thanks for the heads up